c2corg_ui/static/js/auth/auth.js

```diff
@@ -226,7 +226,9 @@ app.AuthController.prototype.register = function() {
   form['lang'] = lang; // inject the current language
 
   this.api_.register(form).then(function() {
-    var msg = alerts.gettext('Register success');
+    var msg = alerts.gettext(
+        'Thank you for your registration! You will receive a confirmation' +
+        'mail in an instance.');
```

Comment:
In an instance?
What about simply "you will soon receive a confirmation email" (or "validation"?)?